README.md

```diff
@@ -62,7 +62,7 @@ Create the following `tslint.json` file like:
 {
   "extends": [ "codelyzer" ],
   "rules":{
-    "angular-whitespace": [true, "check-interpolation", "check-pipe", "check-semicolon"],
```

Comment:
Why are you drop the `check-pipe` rule ? In relation with #412 ?

Reply:
Yes, for now it's dropped only from the readme until we figure it out :-)